### ISSUE TITLE
Accessibility Issue: Aria label missing in question title

### ISSUE THREAD

**Issue by tsv2013** [surveyjs/survey-library#2057]
See the https://surveyjs.answerdesk.io/ticket/details/t3688/accessibility-issue-aria-label-missing thread for more details.
